charts/presto/templates/hive-server-statefulset.yaml

```diff
@@ -80,6 +80,10 @@ spec:
           mountPath: /hadoop/dfs/name
         - name: datanode-empty
           mountPath: /hadoop/dfs/data
+{{- if .Values.spec.config.sharedVolume.enabled }}
+        - name: hive-warehouse-data
+{{ .Values.spec.config.sharedVolume.mountPath }}
```

Comment:
Still not correct. You need to have `path: {{ }}` and have it indented correctly. You should be testing with `spec.config.sharedVolume.enabled=true ` to see what these look like when enabled